### ISSUE TITLE
Global 'issue workflow" structure for kanban columns

### ISSUE THREAD

**Issue by KlavsKlavsen** [go-gitea/gitea#26543]
### Feature Description

To enable more easy migration of issues between projects - and avoid wasting DB space on duplicate data - gitea should do as Jira does.. map "kanban columns" - ie. issue states into a "workflow" - and then one can assign same workflow to multiple projects/kanban boards - and that would also make f.ex.showing a global kanban board much more workable - as those projects using the same workflow - would actually work much better - see https://github.com/go-gitea/gitea/issues/26456 for that usecase.

So in short - to ease issue migration (of state) and also to make it easier to get "correct kanban setup" for new projects, and enabling things like showing things "across multiple projects" - where it would be useful to find issues across projects "in same state" (which they aren't today - even if they're named the same in each project AFAIK ?

**Comment by revoio** [comment]
+1
